dist/images/ovnkube.sh

```diff
@@ -410,8 +547,36 @@ echo "================== ovnkube.sh ================"
 # ovn-node       - all nodes
 
   case ${cmd} in
-    "start_ovn")
-	start_ovn
+    "ovs-server")
+        display_version
+        display_env
+        ovs-server
+    ;;
+    "ovn-northd")
+        display_version
+        display_env
+        setup_cni
```

Comment:
I think you only want `setup_cni` once, for `ovn-node`

Reply:
Changed